content/_index.md

```diff
@@ -24,7 +24,7 @@ imagealt = "Website banner of PyLadiesCon 2024 Online. 6th-8th December | Multi-
           <a href="https://www.linkedin.com/company/pyladiescon"><i class="fab fa-linkedin fa-2x px-2" aria-hidden="true"></i></a>
         </div>
         <div class="w-50 mx-auto mt-2">
```

Comment:
Maybe let's just remove this whole section?

Reply:
I would rather keep the closed message in case someone was wondering if they can still submit. It save the time to click and go to Pretalx